templates/style/_themes.scss

```diff
@@ -32,10 +32,10 @@ html {
   --chart-grid: #ddd;
 }
 
-// To add a new theme, copy the above theme into a new `html[data-theme="name"]`
+// To add a new theme, copy the above theme into a new `html[data-docs-rs-theme="name"]`
 // block below and change the colors
 
-html[data-theme="dark"] {
+html[data-docs-rs-theme="dark"] {
```

Comment:
I don't understand this change, can you explain please?

Reply:
Since https://github.com/rust-lang/rust/pull/115829 rustdoc is using `data-theme` for its own purposes. We might not always have exactly the same themes as rustdoc (e.g. custom themes or lag between changing rustdoc's themes and docs.rs') so we don't want to be using the same variable.

Reply:
Then does that mean that if I change the rustdoc theme, it won't be applied to docs.rs anymore?

Reply:
No, we detect that through the `storage` event and update the docs.rs theme, this is how it has worked since before rustdoc started using `data-theme`.

Reply:
I see. Then sounds good to me.